libs/application-generic/src/usecases/workflow/get-workflow-by-ids/get-workflow-by-ids.dto.ts

```diff
@@ -0,0 +1,8 @@
+import { NotificationTemplateEntity } from '@novu/dal';
+import { WorkflowPreferences } from '@novu/shared';
+
+export class GetWorkflowByIdsResponseDto extends NotificationTemplateEntity {
```

Comment:
```suggestion
export class WorkflowResponseDto extends NotificationTemplateEntity {
```

Reply:
I renamed to `WorkflowInternalResponseDto` to avoid a conflict with the V2 Workflow `WorkflowResponseDto`. Let me know if you think that's acceptable.